src/js/edu-benefits/1990e/config/form.js

```diff
@@ -94,33 +96,33 @@ const formConfig = {
       }
     },
     sponsorVeteran: {
-      title: 'Sponsor Veteran',
+      title: 'Sponsor Information',
       pages: {
         sponsorVeteran: {
-          title: 'Sponsor Veteran',
+          title: 'Sponsor Information',
           path: 'sponsor-veteran',
           uiSchema: {
             veteranFullName: {
               first: {
-                'ui:title': 'Veteran first name'
+                'ui:title': 'Sponsor first name'
```

Comment:
Could we import the common fullName uiSchema definition and override the labels (with _.merge)? That way we don't have to specify that class name again (and makes it easier if we have common changes later).